app/assets/javascripts/discourse/controllers/about.js.es6

```diff
@@ -1,6 +1,10 @@
 import ObjectController from 'discourse/controllers/object';
 
 export default ObjectController.extend({
+  faqOverriden: function() {
+    return !(this.siteSettings.faq_url.length === 0);
```

Comment:
Couldn't this be done as `Ember.computed.gt('siteSettings.faq_url.length', 0)`? I think that should work.